app/main.py

```diff
@@ -2,4 +2,67 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: list[int] = None
+    ) -> None:
+        self.name = name
+        self.weight = weight
+        self.coords = [0, 0] if coords is None else coords
+
+    def go_forward(self, number: int = 1) -> None:
+        self.coords[1] += number
+
+    def go_back(self, number: int = 1) -> None:
+        self.coords[1] -= number
+
+    def go_right(self, number: int = 1) -> None:
+        self.coords[0] += number
+
+    def go_left(self, number: int = 1) -> None:
+        self.coords[0] -= number
+
+    def get_info(self) -> str:
+        return f"Robot: {self.name}, Weight: {self.weight}"
+
+
+class FlyingRobot(BaseRobot):
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: list[int] = None
+    ) -> None:
+        super().__init__(name=name, weight=weight)
+        self.coords = [0, 0, 0] if coords is None else coords
```

Comment:
There is no need to redefine self.coords after intialization, perform assignment in the `super().__init(`
```suggestion
        super().__init__(name=name, weight=weight, coords=coords or [0, 0, 0])
```